taskweaver/code_interpreter/code_generator/code_generator.py

```diff
@@ -115,13 +115,14 @@ def configure_verification(
         self,
         code_verification_on: bool,
         allowed_modules: Optional[List[str]] = None,
+        blocked_functions: Optional[List[str]] = None,
```

Comment:
suggest put in a dict for future extension

Reply:
leave it for future work as currently we only have two configs.